### PR TITLE
Add Gothenburg Sweden to the list of hometowns

### PR DESCRIPTION
**Pull request by wangjingee**
- Anyone been to Gothenburg in Sweden before
- Anything to add, amend or remove (highly appreciated)
- Do not dare to AT (@) anyone as I do not know any of you

### CODE REVIEW

File: wangjingee-Gothenburg.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Attractions
+- Aveny
+- Kanalen
+- Tradgardsforening
```
Comment (howdeed) [inline wangjingee-Gothenburg.md:4]: What are these attractions?

**Comment by wangjingee** [comment]
@howdeed Please care to comment? Thanks in advance!

**Comment by wangjingee** [comment]
@howdeed The first one is the main road; second one a river / canal; third one a beautiful park

File: wangjingee-Gothenburg.md (hunk 1 above)
Comment (sunyinz) [inline wangjingee-Gothenburg.md]: What is this attraction about?

**Pull request merged**